tests/test_utils.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import pytest
+from pandas import DataFrame
+
+from bulk.utils import get_color_mapping
```

Comment:
The current error seems to make a bit of sense. Your test is importing from `bulk.utils` but I think it needs to import from `bulk._bokeh.utils`.